install.sh

```diff
@@ -341,12 +341,12 @@ do_install() {
 			esac
 		;;
 
-		centos|rhel)
+		centos|rhel|sles)
 			if [ -z "$dist_version" ] && [ -r /etc/os-release ]; then
 				dist_version="$(. /etc/os-release && echo "$VERSION_ID")"
 			fi
 		;;
-
+		
```

Comment:
Looks like you accidentally added some whitespace here; I spotted some other locations where whitespace was added; let me quickly push a PR with those removed, so that we can get this merged